kernelflinger.c

```diff
@@ -1200,7 +1200,8 @@ EFI_STATUS efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *sys_table)
 	InitializeLib(image, sys_table);
 
 #ifdef USE_UI
-	ux_display_vendor_splash();
```

Comment:
use this macro
#ifndef USB_SBL
    // disable for now to save boot time
    ux_display_vendor_splash();
#endif